engine/src/operators/OrderBy.h

```diff
@@ -17,9 +17,11 @@ std::unique_ptr<ral::frame::BlazingTable> sort(const ral::frame::BlazingTableVie
 
 std::unique_ptr<ral::frame::BlazingTable> sample(const ral::frame::BlazingTableView & table, const std::string & query_part);
 
-std::unique_ptr<ral::frame::BlazingTable> generate_distributed_partition_plan(const ral::frame::BlazingTableView & selfSamples, std::size_t table_num_rows, std::size_t avg_bytes_per_row, const std::string & query_part, Context * context);
+std::unique_ptr<ral::frame::BlazingTable> generate_distributed_partition_plan(const ral::frame::BlazingTableView & selfSamples, 
```

Comment:
is there a reason to have two apis and not a flag with a single api that toggles beween two private implementations?

Reply:
I mention this because they have the exact same signature.

Reply:
they dont have the same API. One takes in `ral::frame::BlazingTableView & selfSamples` the other `std::vector<ral::frame::BlazingTableView> & samples`. 
Additionally `generate_distributed_partition_plan` handles distributed stuff and calls `generate_partition_plan`